src/emulator/hubExport.ts

```diff
@@ -74,6 +88,43 @@ export class HubExport {
     });
   }
 
+  private async exportDatabase(metadata: ExportMetadata): Promise<void> {
+    const databaseInfo = EmulatorRegistry.get(Emulators.DATABASE)!.getInfo();
+    const databaseAddr = `http://${databaseInfo.host}:${databaseInfo.port}`;
+
+    // Make sure the export directory exists
+    if (!fs.existsSync(this.exportPath)) {
+      fs.mkdirSync(this.exportPath);
+    }
+
+    const dbExportPath = path.join(this.exportPath, metadata.database!.path);
+    if (!fs.existsSync(dbExportPath)) {
+      fs.mkdirSync(dbExportPath);
+    }
+
+    // Get the list of namespaces
+    const inspectURL = `http://${databaseInfo.host}:${databaseInfo.port}/.inspect/databases.json?ns=${this.projectId}`;
+    const inspectRes = await api.request("GET", inspectURL);
+
+    for (const instance of inspectRes.body) {
+      logger.debug(`Exporting database instance: ${instance}`);
+
+      const ns = instance.name;
+      const url = `${databaseAddr}/.json?ns=${ns}`;
```

Comment:
Please also add [`?format=export`](https://firebase.google.com/docs/reference/rest/database#section-param-format) to preserve `.priority`.